docs/support_playbook.md

```diff
@@ -106,3 +114,61 @@ status: 400, body: {"title":"Teacher has no incomplete ITT record","status":400,
 * Cross reference the trainee details on Register with the trainee details on DQT, you can use the DQT API for this - checking the trainee timeline on Register can also be helful
 * If there are no differences, it is likely a race condition and you can delete the failed update job
 * If there are differences, speak to DQT and maybe contact provider to see what needs updating
+
+### Dqt::RetrieveTrnJob for Trainee id: xxx has timed out after 4 days.
+
+We see this error on slack when our polling job times out before we receive a
+TRN from DQT. The default time out is four days from the trainee's
+`submitted_for_trn_at` date.
+
+We need to understand from DQT why the TRN is taking a while to assign. Is there
+a problem we need to address?
+
+If there is no issue, we can kick off another polling job with a timeout in the
+future:
+
+```ruby
+Dqt::RetrieveTrnJob.perform_later(t.dqt_trn_request, Date.today + 4.days)
+```
+
+### Dqt::TraineeUpdate::TraineeUpdateMissingTrn
+
+This error will be accompanied by the message:
+
+`Cannot update trainee on DQT without a trn (id: 142508)`
+
+This means that we have tried to update a trainee on DQT before we've received a
+TRN back.
+
+Two issues are possible:
+
+1. The trainee is not present on DQT for some reason (it may have failed
+  previously with one of the above errors).
+
+    In this case, we need to re-register them:
+
+    ```ruby
+    Dqt::RegisterForTrnJob.perform_later(t)
+    ```
+
+    making sure that it succeeds.
+
+    We also need to kick off another polling job, so that we receive the TRN
+    when assigned:
+
+    ```ruby
+    Dqt::RetrieveTrnJob.perform_later(t.dqt_trn_request, Date.today + 4.days)
```

Comment:
so the polling job will start in 4 days time, and run for 4 days (or until is retrieves the TRN)?

Reply:
the polling job starts immediately and will timeout in 4 days if it doesn't receive a TRN. At that point we get the `Dqt::RetrieveTrnJob for Trainee id: xxx has timed out after 4 days.` message in Slack.